lib/notifications/open.dart

```diff
@@ -95,6 +98,24 @@ class NotificationOpenManager {
       // TODO(#82): Open at specific message, not just conversation
       narrow: openData.narrow);
   }
+
+  /// Navigates to the [MessageListPage] of the specific conversation
+  /// for the provided payload that was attached while creating the
+  /// notification.
+  Future<void> _navigateForNotification(NotificationPayloadForOpen payload) async {
```

Comment:
Another reason the `NotificationPayloadForOpen` rename would be helpful: currently, it's pretty unclear that this private method is only about notifications that come while the app is open. It'll be helpful for debugging if it's easier to see what code is about the launch notification vs. not.